src/main/java/com/thealgorithms/maths/HeronsFormula.java

```diff
@@ -1,18 +1,27 @@
 package com.thealgorithms.maths;
 
 /**
+ * Wikipedia for HeronsFormula => https://en.wikipedia.org/wiki/Heron%27s_formula
  * Find the area of a triangle using only side lengths
  */
 
-public class HeronsFormula {
+public final class HeronsFormula {
 
-    public static double Herons(int s1, int s2, int s3) {
-        double a = s1;
-        double b = s2;
-        double c = s3;
-        double s = (a + b + c) / 2.0;
-        double area = 0;
-        area = Math.sqrt((s) * (s - a) * (s - b) * (s - c));
-        return area;
+    /*
+     * A function to get the Area of a Triangle using Heron's Formula
+     * @param s1,s2,s3 => the three sides of the Triangle
+     * @return area using the formula (√(s(s – s1)(s – s2)(s – s3)))
+     * here s is called semi-perimeter and it is the half of the perimeter (i.e; s = (s1+s2+s3)/2)
+     * @author satyabarghav
+     */
+    private HeronsFormula() {
+    }
+    public static double Herons(final double a, final double b, final double c) {
```

Comment:
Using this opportunity, I would suggest to rename the method to start with the _small_ `h`.

```suggestion
    public static double herons(final double a, final double b, final double c) {
```